io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl

```diff
@@ -0,0 +1,417 @@
+#######################################################################
+# Copyright (c) 2023 BMW AG
+# Copyright (c) 2023 BASF Coatings GmbH
+# Copyright (c) 2023 SAP Deutschland SE & Co.KG
+# Copyright (c) 2023 T-Systems International GmbH
+# Copyright (c) 2023 Robert Bosch GmbH
+# Copyright (c) 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.secondary_material_content:1.0.0#> .
+@prefix ext-part: <urn:samm:io.catenax.serial_part:1.0.1#> .
+
+:SecondaryMaterialContent a samm:Aspect ;
+   samm:preferredName "Secondary Material Content"@en ;
+   samm:description "This model refers to a part containing one or more different material. It references the share on primary and secondary material and relevant details."@en ;
+   samm:properties ( :secondaryMaterialContent :supplierID :orderNumber ext-part:manufacturerPartId ext-part:customerPartId ext-part:nameAtManufacturer ext-part:nameAtCustomer ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:secondaryMaterialContent a samm:Property ;
+   samm:preferredName "secondary Material Content"@en ;
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
+   samm:characteristic :SecondaryMaterialContentMaterials .
+
+:supplierID a samm:Property ;
+   samm:preferredName "supplier ID"@en ;
+   samm:description "Suppliers individual business ID within the Catena-X network."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "BPN1234567890ZZ" .
+
+:orderNumber a samm:Property ;
+   samm:preferredName "order Number"@en ;
+   samm:description "Order number as provided by supplier."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "856496" .
+
+:SecondaryMaterialContentMaterials a samm-c:Set ;
+   samm:preferredName "Secondary Material Content Materials"@en ;
+   samm:description "List of relevant materials which are built in this part and details on the share of primary and secondary material."@en ;
```

Comment:
check for singular and plural use of material. I am also not sure, whether this is some domain specific use of the term but for me it sounds like primary and secondary are two materials thus would opt for materials. But maybe I do not get that right